src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -211,6 +211,19 @@ export class AddEditProjectComponent implements OnInit, OnDestroy {
     return this.myForm.get('agreements') as FormArray;
   }
 
+  launchFilePicker() {
```

Comment:
This file could use better docs for sure, however if you add a function, please at minimum add something to it.